compiler/one-cmds/tests/one-infer_006.py

```diff
@@ -0,0 +1,14 @@
+# This script gets one argument and print it
```

Comment:
how about renaming this as `one-infer-test-post-process.py` ?

Reply:
That's great 👍